docs/astro/src/content/docs/guide/platforms/web.mdx

```diff
@@ -0,0 +1,101 @@
+---
+<!-- Copyright © SixtyFPS GmbH <info@slint.dev> ; SPDX-License-Identifier: MIT -->
+title: Web
+description: Running Slint on the Web with Wasm
+---
+
+While Slint is mainly designed for native applications (desktop, embedded, mobile),
+it can also run in a web browser using WebAssembly (Wasm).
+
+:::caution[Caution]
+Currently, only the Rust language is supported when targeting the web, and the only renderer available is FemtoVG.
+:::
+
+This renderer draws the UI into a `<canvas>` element on the page.
+The rendering happens entirely within the canvas, so Slint does not rely on the browser's DOM or CSS engine.
+This makes it possible to have a consistent look across platforms, but also introduces some limitations:
+
+* Slint handles its own font rendering, so it does not benefit from the browser's text rendering.
+* Accessibility features (such as screen readers) are not available.
+* The UI does not behave like a typical web application, since it doesn't use standard HTML elements.
```

Comment:
```suggestion
* Slint renders text directly, instead of benefitting from the browser's text rendering.
* Accessibility features (such as screen readers) are not available.
* The UI does not behave like a typical web application, since it doesn't use standard HTML elements.
```